code/worker.py

```diff
@@ -1,38 +1,40 @@
 #!/usr/bin/env python3
+import argparse
+
+import requests
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'appimage_url', metavar='URL',
+        help="URL of the AppImage file to query")
+    ns = parser.parse_args()
+    r = requests.get(ns.appimage_url, stream=True)
```

Comment:
The `r` is a stock request style but I reamed it to `req`. I didn't add the newline (or below) because this is just too small for one and it breaks the flow of a trivial function. Not saying none are needed eventually but then we'll probably split this out of main anyway.